Diagnostics/PlotLLH.cpp

```diff
@@ -536,8 +536,8 @@ int PlotLLH(){
             splitSamplesRatioPad = new TPad("splitSampRatioPad", "splitSampRatioPad",0.0 ,0.0 , 0.0, 0.0);
           }
 
-          std::vector<Double_t> cumSums;
-          std::vector<bool> drawLabel;
+          std::vector<Double_t> cum_Sums;
```

Comment:
Do we have any standard formatting of variable names moving forward or is this just personal tidy? 😄

Reply:
No this related with Shadowing.

I have added shadow flag and in
https://github.com/mach3-software/MaCh3/blob/7a49c183318b118b4208fb70cb6c29f754160b1b/Diagnostics/PlotLLH.cpp#L492C35-L492C38
you had cumSums.

This isn't error but I am trying to avoid shadowing, as I think this way code will be more readable by humans.